yang/ietf-l2vpn-ntw.yang

```diff
@@ -726,6 +731,58 @@ module ietf-l2vpn-ntw {
     }
   }
 
+  //
+
+  augment /l2vpn-ntw:l2vpn-ntw/l2vpn-ntw:vpn-services/l2vpn-ntw:vpn-service/l2vpn-ntw:vpn-nodes/l2vpn-ntw:vpn-node/l2vpn-ntw:signaling-options/l2vpn-ntw:signaling-option/l2vpn-ntw:bgp/l2vpn-ntw:vpn-target {
```

Comment:
I don't think this can be handled as an augment of a data node in the same module. 

BTW, are we trying to achieve this https://datatracker.ietf.org/doc/html/rfc7432#section-7.10.1?

Wouldn't that be internal to the controller once the Ethernet Tag ID (VLAN ID) is assigned?

Reply:
I was trying to augment the grouping of the VPN common, but it did not work. I thought the augment was a good option to not completely redefine the Route Targets.

Yes, we are trying to achieve the https://datatracker.ietf.org/doc/html/rfc7432#section-7.10.1. 

In my undestanding the VLAN ID , is equivalent to the ESI:

>With this service interface, an EVPN instance consists of only a
   single broadcast domain (e.g., a single VLAN).  Therefore, there is a
   one-to-one mapping between a VID on this interface and a MAC-VRF.